tests/unit/dataactvalidator/test_b20_object_class_program_activity.py

```diff
@@ -17,28 +17,42 @@ def test_column_headers(database):
 
 def test_success(database):
     """ Tests that all combinations of TAS, program activity code, and object class in File C exist in File B """
-    tas = "".join([_TAS, "_success"])
-
-    op = ObjectClassProgramActivity(job_id=1, row_number=1, tas=tas, program_activity_code='1', object_class='1')
```

Comment:
Thanks for removing the extraneous bits here and leaving what's necessary--easier to grok this way.